scripts/scraper-ng/rules/html-require-recipe-ingredients/index.js

```diff
@@ -1,10 +1,11 @@
 const fs = require("fs");
 const path = require("path");
-
 const yaml = require("js-yaml");
```

Comment:
Side note here: I've been grouping imports by standard library, external libraries, and then local modules. I don't know if we should have a style for this sort of thing, but that's what I was doing.

Reply:
Oh I didn't realise that, but I'm happy for us to to this. I've put the blank line back :).